aptos-move/aptos-validator-interface/src/lib.rs

```diff
@@ -73,7 +73,7 @@ pub trait AptosValidatorInterface: Sync {
         )>,
     >;
 
-    async fn get_latest_version(&self) -> Result<Version>;
+    async fn get_committed_version(&self) -> Result<Version>;
```

Comment:
IMO, committed_version just means the version that already written to db. I think here this is get_ledger_committed_version ?

Reply:
outside of the DB code it's pretty clear? and there's parity between "synced version" and "committed version"

Reply:
maybe get_latest_ledger_info_version()?

Reply:
> maybe get_latest_ledger_info_version()?

I like this.